static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -47,13 +49,14 @@ define([
      * Trigger sticky scrolling for MPUs in the right-hand article column
      */
     sizeCallbacks[adSizes.mpu] = function (_, advert) {
-        if (advert.node.classList.contains('ad-slot--right')) {
+        var $node = bonzo(advert.node);
+        if ($node.hasClass('ad-slot--right')) {
             var mobileAdSizes = advert.sizes.mobile;
             if (mobileAdSizes && mobileAdSizes.some(function (size) { return size[0] === 300 && size[1] === 251; })) {
-                stickyMpu(bonzo(advert.node));
+                stickyMpu($node);
             }
-        } else if (advert.node.classList.contains('ad-slot--facebook')) {
-            advert.node.classList.add('ad-slot--fluid');
+        } else if ($node.hasClass('ad-slot--facebook')) {
+            $node.addClass('ad-slot--fluid');
```

Comment:
does this need to be fast-dom'd? if it's called from fd, there's a lot of calc in there that doesn't need to be?